src/env.ts

```diff
@@ -57,8 +64,8 @@ export const envConfig = {
   DB_PORT: envalid.port({ default: 5432 }),
   COOKIE_SESSION_KEYS: strArrayValidator({ devDefault: ['secret'] }),
   PUBLIC_URL: envalid.url({ devDefault: 'http://localhost:3000' }),
-  API_SWAGGER_BG_COLOR: envalid.str({ default: '#fafafa' }),
-  API_SWAGGER_TITLE: envalid.str({ default: 'Veritable' }),
+  API_SWAGGER_BG_COLOR: envalid.str({ devDefault: '#ff3131', default: '#ffffff' }),
+  API_SWAGGER_TITLE: envalid.str({ devDefault: 'Bob', default: 'veritable' }),
```

Comment:
Why add these devDefaults?

Reply:
for demo:) since demo mode default is true, and we setup as alice initially anyway

Reply:
updated, thanks for a flag up.

Reply:
so this is the one that's meant to be 'Alice'?

Reply:
Indeed, has been updated